kalite/distributed/static/js/distributed/bundle_modules/base.js

```diff
@@ -14,6 +14,9 @@ require("browsernizr/test/canvas");
 require("browsernizr/test/touchevents");
 var Modernizr = require("browsernizr");
 
+// Expose this as a global object for use in central server inline JS.
+global.getCookie = require("utils/get_cookie");
+
```

Comment:
This needs to be configurable and only switched on in the central server.

Reply:
Okay, this might actually be harmless to the rest of the KA Lite application, wouldn't you think so @rtibbles ?